versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx

```diff
@@ -524,7 +524,7 @@ Example
 http://localhost:31986/productpage
 ```
 The following is an example of the product page:
-![alt](/images/Bookinfo-productpage-OS.png)
+![alt](/img/Bookinfo-productpage-OS.png)
```

Comment:
The same comment as above for changing `images` to `img`, and this change holds good to all instances where `images` is changed to `img`.